testing/supporting_docs/appium_information.md

```diff
@@ -0,0 +1,69 @@
+## Appium 
+Appium is a cross platform UI testing tool for mobile applications running on 
+either android or iOS devices.  It can be used to test native applications, 
+hybrid applications like Cordova and Xamarin, and web app testing through mobile 
+device browsers.  One of major points of the Appium philosophy is to avoid time 
```

Comment:
@twnolan there is an unnecessary double space at the beginning of the sentence. 'major' might be replaced by 'the key'